### PR TITLE
Update tasmota.markdown

### PR DESCRIPTION
**Pull request by Micha-Btz**
Add hint for discovery from https://tasmota.github.io/docs/Home-Assistant/

## Proposed change
Add a hint to avoid mistakes ( As I did)

## Type of change
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).

## Checklist
- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @Micha-Btz, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#17562).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Comment by emontnemery** [comment]
@Micha-Btz I would expect the Tasmota integration to work also if you have changed the fulltopic. Can you comment on what fulltopic didn't work?

**Comment by Micha-Btz** [comment]
I have had set topic and full topic to Blitzwolf_Decke and the discovery from HA is not working anymore.
If I change full topic back to %prefix%/%topic%/ it works. Maybe %prefix%/ would be enough.

**Comment by emontnemery** [comment]
Yes, that's an invalid topic, there needs to a placeholder at least for the `%prefix%`, although it's not super clear from the documentation: https://tasmota.github.io/docs/MQTT/#fulltopic

Maybe the change in this PR can be rephrased to something like:
> The Tasmota integration supports custom [`fulltopic`](https://tasmota.github.io/docs/MQTT/#fulltopic), but it is strongly suggested to leave `fulltopic` at its default setting as Tasmota does not prevent setting an invalid or non-unique fulltopic, for example a fulltopic without the `%prefix%` or `%topic%` tokens.

**Comment by Micha-Btz** [comment]
I have update it, feel free to suggest changes, since English is not my main language.

**Review by emontnemery — APPROVED** [review]

File: source/_integrations/tasmota.markdown [hunk 1]
```diff
@@ -31,6 +31,7 @@ This integration allows you to control [Tasmota](https://tasmota.github.io/docs/
 - Tasmota devices' MQTT setting configured to communicate with the MQTT broker.
 - Tasmota devices flashed with version 9.2, or later.
 - Tasmota devices configured for native discovery (`SetOption19 0`)
+- Although the Tasmota integration supports custom fulltopic it is strongly suggested to leave fulltopic at its default, Tasmota does not prevent setting an invalid or non-unique fulltopic, for example a fulltopic without the %prefix% or %topic% tokens.
```
Comment (frenck) [inline source/_integrations/tasmota.markdown:34]:
```suggestion
- Although the Tasmota integration supports custom full topic it is strongly suggested to leave full topic at its default, Tasmota does not prevent setting an invalid or non-unique full topic, for example, a full topic without the %prefix% or %topic% tokens.
```

Comment (emontnemery) [reply source/_integrations/tasmota.markdown:34]: "fulltopic" refers to a Tasmota setting, writing it as "full topic" would be consfusing. We could quote it instead as `fulltopic`

**Review by frenck — APPROVED** [review]

**Pull request merged**